README.md

```diff
@@ -17,7 +17,7 @@ The Mapbox Route Visualizer is a powerful bash script that creates a visually ap
 - curl
 - jq (`brew install jq`)
 - ImageMagick (`brew install imagemagick`)
-- Mapbox account and access token
+- Mapbox account and access token (create a private one with _all_ scopes)
```

Comment:
maybe you know which scopes are required—I tried initially w/ the tileset and styles ones but ultimately just did _everything_ and it worked